src/fides/core/config/utils.py

```diff
@@ -4,6 +4,29 @@
 DEFAULT_CONFIG_PATH_ENV_VAR = "FIDES__CONFIG_PATH"
 
 
+def replace_config_value(
```

Comment:
This is a semi-hacky workaround. Because the new config file that we export is more than just a plain `dict` -> `toml`  string, it makes more sense to do string replacement for updating config values. This shouldn't need to be used often